nilearn/maskers/multi_nifti_masker.py

```diff
@@ -309,6 +309,9 @@ def fit(
 
             self.mask_img_ = check_niimg_3d(self.mask_img)
 
+            # Just check that the mask is valid
+            masking.load_mask_img(self.mask_img_)
```

Comment:
How about directly importing `load_mask_img`?

Reply:
will refactor to just import the required functions in a follow up pr